### ISSUE TITLE
the screen reader doesn't read text on screen like label's and other images, It only reads interactive elements like buttons.

### ISSUE THREAD

**Issue by ravi1124339** [xamarin/Xamarin.Forms#1946]
### Steps to Reproduce

1. Create a label and provide AutomationProperties
2. Start the narrator for Windows 10 or JAWS 18
3. Run the project on Local Machine.

### Expected Behavior
The narrator should read the text written in the Label, Image or any other non interactive element if AutomationProperties are set in that control.

### Actual Behavior
The narrator is ommiting the text written in the Label, Image or any other non interactive element.

### Basic Information

- Version with issue: 2.5.0.280555, 2.3.4.231, 2.4.0.282 (checked with these three versions)
- Last known good version: none of the provided stable version are working with narrator
- IDE: Visual Studio 2015 update 3
- Platform Target Frameworks: Currently working with UWP only
  - UWP:  Windows  App Target vesion Windows 10(10.0; build 10240)
- Android Support Library Version: 
- Nuget Packages: Xamarin Forms

###Code Snippet used is attached in the file attached
[Accessibility.txt](https://github.com/xamarin/Xamarin.Forms/files/1746927/Accessibility.txt)
